src/main/java/org/opentripplanner/apis/transmodel/model/framework/PenaltyForStreetModeType.java

```diff
@@ -1,6 +1,4 @@
-package org.opentripplanner.ext.transmodelapi.model.framework;
-
-import static org.opentripplanner.ext.transmodelapi.model.EnumTypes.STREET_MODE;
```

Comment:
Is the removal of static import of enum constants intended or a side-effect of the refactoring move?

I personally would prefer to keep the static imports, but for this and the cases below it doesn't seem to matter that much. Generally though, our enums seems be readable on their own.

Reply:
I have noticed that IntelliJ does this when refactoring. I believe it is to not end up with a naming conflict after the move (witch is possible). This is to minor that I will spend time fixing it - hope it is ok.

Reply:
Yes, no problem.